desktop/apps/artwork/components/artists/helpers.coffee

```diff
@@ -15,24 +15,21 @@ module.exports =
   sortExhibitions: (shows) ->
     sortBy(shows, 'start_at').reverse()
 
-  sections: sections = (artist, hasMarketDataEnabled) ->
+  sections: sections = (artist) ->
     has: (section) ->
       switch section
         when 'biography'
           artist.blurb || artist.bio || artist.biography_blurb?.text != ""
         when 'exhibition_highlights'
-          if hasMarketDataEnabled
-            artist.exhibition_highlights? && artist.exhibition_highlights.length > 0
-          else
-            artist.exhibition_highlights? && artist.exhibition_highlights.length > 15
+          artist.exhibition_highlights?.length > 0
```

Comment:
b/c we're using `?` existential operator we don't need two separate conditionals

Reply:
:+1: